Requesting Signoff for Sign

Aaron-Wilhite opened the pull request:
Requesting signoff for sign

Aaron-Wilhite committed:
Requesting Signoff for Sign

Aaron-Wilhite committed:
Added BOM to Sign Signoff

Aaron-Wilhite committed:
Add files via upload

Aaron-Wilhite committed:
Add files via upload

Aaron-Wilhite committed:
Delete LED Strip to Arduino Diagram.dwg

Aaron-Wilhite committed:
Add files via upload

Aaron-Wilhite committed:
Delete Sign Buildable Schematic.pdf

Aaron-Wilhite committed:
Add files via upload

Aaron-Wilhite committed:
Update Sign

Aaron-Wilhite committed:
Rename Sign to Sign.md

Aaron-Wilhite committed:
Update Sign.md

Aaron-Wilhite committed:
Update Sign.md

Aaron-Wilhite committed:
Update Sign.md

Commit:
Renamed pictures in Sign Signoff

Aaron-Wilhite committed:
Update Sign.md

JesseTNRoberts commented on Documentation/Signoffs/Sign.md:
```diff
@@ -0,0 +1,96 @@
+
+
+
+## Function of the Subsystem
```
Fix punctation etc.

JesseTNRoberts commented on Documentation/Signoffs/Sign.md:
```diff
@@ -0,0 +1,96 @@
+
+The sign that will be implemented into this project will show the current number of available parking spots for the monitored lot.This will be updated roughly as often as the mobile application. The data being sent to the display will be from the primary and secondary data acquisition systems. That data will then be sent to the server and programmed into readable data to the sign. The sign will t ...
+
+## Constraints
```
Some of these are too ambiguous to be design targets.

JesseTNRoberts replied:
This hasn't been addressed

JesseTNRoberts replied:
change this to a list of constraints with explanation only added where necessary.

JesseTNRoberts commented on Documentation/Signoffs/Sign.md:
```diff
@@ -0,0 +1,96 @@
+3. Outdoor Physical Sign/Stand
+  -Text above Seven-Segment Display Stating “Open Spots Available”
+
+4. 12 V DC Power for LEDs and Arduino
```
Why is the voltage being stated a priori?

JesseTNRoberts commented on Documentation/Signoffs/Sign.md:
```diff
@@ -0,0 +1,96 @@
+
+## Buildable Schematic
+
+![Figure 2. Physical Sign Schematic with LED Strip Formation](../Images/Sign_Buildable_Schematic.PNG)
```
No 3d schematic. The electrical schematic software needs to match across all designs. Discuss as a team which to use. This style is hard to use and does not scale well. 

Most importantly, this would not work (seems unrealistic to think a low power MCU will provide current to LEDs that are visible at a distance). Nor does it deal with any of the powering and communication distance.

Aaron-Wilhite committed:
Delete LED_Strip_to_Arduino_Diagram.PNG

Aaron-Wilhite committed:
Delete Arduino_to_LED_and_Wifi_Module.PNG

Aaron-Wilhite committed:
Display Schematic Added

Aaron-Wilhite committed:
Delete Sign_Buildable_Schematic.PNG

Aaron-Wilhite committed:
Add 3D Model of Sign

Aaron-Wilhite committed:
Add 3D Model of Sign

Aaron-Wilhite committed:
Delete Display_Schematic.pdf

Aaron-Wilhite committed:
Add Display Schematic

Aaron-Wilhite committed:
Update Sign.md

Aaron-Wilhite committed:
Update Sign.md

Aaron-Wilhite committed:
Add files via upload

Aaron-Wilhite committed:
Update Sign.md

JesseTNRoberts commented on Documentation/Signoffs/Sign.md:
```diff
@@ -0,0 +1,90 @@
+## Constraints
+
+1. Seven-Segment Display & Controls
+  -At least 1 foot in height and 6 inches in width (LED display)
```
Why? What is the reasoning behind this constraint?

JesseTNRoberts commented on Documentation/Signoffs/Sign.md:
```diff
@@ -0,0 +1,90 @@
+
+The Seven-Segment (SS) display needs to be large enough to be seen from the entrance into the parking lot. After lots of research, not many options are available for large seven segment displays, so the idea of creating a custom made large waterproof seven segment display came to light. This will consist of seven red 12 V LED light strips [2] per display, so 14 of them total. The chosen LED strip ...
+
+![Figure 2. Arduino Circuit Diagram](../Electrical/Schematics/Sources/Display_Schematic.PNG)]
```
This isn't buildable as there are a number of components that don't have part numbers or labels. Also, it is best if the different designs are all done in the same software. (Ideally) 

But it can be ok in this case to use a different software as this won't have electrical connections to the other systems.

JesseTNRoberts reviewed (commented):
There is really no significant analysis of the design or constraints.

Aaron-Wilhite committed:
Delete 3DSign.PNG

Aaron-Wilhite committed:
Update 3D Sign

Aaron-Wilhite committed:
Delete 3DSign.PNG

Aaron-Wilhite committed:
Delete 3DSignPDF.pdf

Aaron-Wilhite committed:
Update 3D Sign

Aaron-Wilhite committed:
Delete 3DSign.PNG

Aaron-Wilhite committed:
Update Sign

JesseTNRoberts reviewed (commented):
Previous comments have not all been addressed

Aaron-Wilhite committed:
Update Sign.md

Aaron-Wilhite committed:
Delete Display_Schematic.PNG

Aaron-Wilhite committed:
Delete Display_Schematic.pdf

Aaron-Wilhite committed:
Add Display Schematic

JesseTNRoberts reviewed (commented):
Your sign schematic is not buildable. It does not come with LED strip mounting mechanisms nor does it have cutouts for them. How is the text going to be placed on the sign?

Aaron-Wilhite committed:
Update Sign.md

Aaron-Wilhite committed:
Update Sign.md

Aaron-Wilhite committed:
Update Sign.md

Aaron-Wilhite committed:
Update Sign.md

JesseTNRoberts reviewed (commented):
The constraints need to be listed. Paragraphs aren't appropriate. If you need additional space to explain something, you can add a background section. 

This is a design document. State the constraints and the specs (and if where they come from is not clear, explain in background), give the schematic, analyze it to show it meets the constraints, give the BOM for all components in the schematic. There is no reason for lengthy paragraphs.

Aaron-Wilhite committed:
Update Sign.md

Aaron-Wilhite committed:
Add V2nd View of 3D Schematic

Shows view of 3D schematic with mounted weatherproof case

Aaron-Wilhite committed:
Update Sign.md

Aaron-Wilhite committed:
Update Sign.md

Aaron-Wilhite committed:
Update Sign.md

JesseTNRoberts reviewed (commented):
Did not show that the LEDs would be visible in daylight.

Aaron-Wilhite committed:
Update Sign.md

Changes the overall design of the physical sign

Aaron-Wilhite committed:
Update Sign.md

Aaron-Wilhite committed:
Delete 3DSign.PNG

Aaron-Wilhite committed:
Delete 3DSignView2.PNG

Aaron-Wilhite committed:
Delete Sign Buildable Schematic.dwg

Aaron-Wilhite committed:
Add New Sign Models

Aaron-Wilhite committed:
Update Sign.md

Aaron-Wilhite committed:
Update Sign.md

JesseTNRoberts commented on Documentation/Signoffs/Sign.md:
```diff
@@ -0,0 +1,180 @@
+## Constraints
+
+1. Seven-Segment Display & Controls
+- Must have waterproof controllable display viewable in the daylight (At least 2000 millicandelas at a 60 degree viewing angle or 1.68 Lumens [12] [13]) and from the road capable of showing number of open parking spots 
```
There's not a citation 12 or 13. I don't think that 1.68 lumens is sufficient. I did see the forum in the citations and I don't think that is an appropriate source of information from which to base a design decision.

JesseTNRoberts reviewed (commented):
This looks good. The only issue I see is that the LED visibility still seems to not be very well analyzed. The other issue is, how do you know that the Mosfets will be able to power the led strips?

Aaron-Wilhite committed:
Update Sign.md

- Redefined LED visibility constraints and analysis
- Clarified transistor constraints, specs, and analysis

Aaron-Wilhite committed:
Update Sign.md

Aaron-Wilhite committed:
Update Sign.md

Aaron-Wilhite committed:
Update Sign.md

Aaron-Wilhite committed:
Update Sign.md

Aaron-Wilhite committed:
Update Sign.md

JesseTNRoberts commented on Documentation/Signoffs/Sign.md:
```diff
@@ -57,12 +55,13 @@ a. LED Strips
 > - Already waterproof (IP67 rated)
-> - These LED strips claim to be rated for 1000 Lumens each, which meets the constraint of at least 2000 millicandelas at a 60 degree viewing angle or 1.68 Lumens to be seen in the daylight. 
+> - LED strips rated for 1000 Lumens each, a max viewing apex angle of 160 degrees (average max for LED strips) equates to roughly 318.3 Candela. Over both seven-segment displays (16 inches x 13 inches) equate to 2,371.5 Nits (Cd/m^2) which meets the constraint of at least 1000 Nits total.
+Note: This is typically used for LED/LCD array signs which require much more clarity for optimal legibility [8]. 
```
8 says nothing about this.

AWilhite42 replied:
I meant to take that out, it was from the previous source, sorry for that mistake

JesseTNRoberts reviewed (commented):
Address 7 segment visibility.

Aaron-Wilhite committed:
Update Sign.md

Aaron-Wilhite committed:
Update Sign.md

Aaron-Wilhite committed:
Update Sign.md

Aaron-Wilhite committed:
Update Sign.md

Aaron-Wilhite committed:
Update Sign.md

JesseTNRoberts commented on Documentation/Signoffs/Sign.md:
```diff
@@ -0,0 +1,200 @@
+
+- Keiurot LED strips chosen for:
+> - Already waterproof (IP67 rated)
+> - LED strips rated for 1000 Lumens each, a max viewing apex angle of 160 degrees (average max for LED strips) equates to roughly 318.3 Candela. Over both seven-segment displays (16 inches x 13 inches) equate to 2,371.5 Nits (Cd/m^2) which meets the constraint of at least 1000 Nits total for daylight visibility. 
```
The article is about displays. You don't have candela per square meter as this is individual LEDs.

JesseTNRoberts replied:
So, I'm still not convinced this will be visible.

JesseTNRoberts reviewed (commented):
Where are the components housed?

Aaron-Wilhite committed:
Delete NewSignView1.PNG

Aaron-Wilhite committed:
Delete NewSignView2.PNG

Aaron-Wilhite committed:
Add files via upload

Aaron-Wilhite committed:
Update Sign.md

Aaron-Wilhite committed:
Delete NewSignView1.PNG

Aaron-Wilhite committed:
Delete NewSignView2.PNG

Aaron-Wilhite committed:
Add files via upload

Aaron-Wilhite committed:
Update Sign.md

JesseTNRoberts reviewed (commented):
Please include excerpts from source 12 supporting your design and conclusions.

Aaron-Wilhite committed:
Update Sign.md

JesseTNRoberts reviewed (commented):
The last issue I see is that the sign is not buildable. 2d line drawings aren't going to be sufficient (unless you are very creative). You need to completely represent how this is to be built.

Aaron-Wilhite committed:
Update Sign.md

Aaron-Wilhite committed:
Delete NewSignView1.0.PNG

Aaron-Wilhite committed:
Add files via upload

Aaron-Wilhite committed:
Update Sign.md

Aaron-Wilhite committed:
Update Sign.md

Aaron-Wilhite committed:
Update Sign.md

Aaron-Wilhite committed:
Update Sign.md

Aaron-Wilhite committed:
Update Sign.md

JesseTNRoberts reviewed (commented):
The words at the top seem as though they will not be readable. Further, it is interesting that you found lenses that are the exact size you need. It looks like you'll have to fabricate those. Which makes it unlikely that there is simple "mounting hardware" for them. If there is mounting hardware, unless it is represented here, how is someone supposed to build this?

Aaron-Wilhite committed:
Update Sign.md

Aaron-Wilhite committed:
Delete NewSignView1.1.PNG

Aaron-Wilhite committed:
Delete NewSignView2.0.PNG

Aaron-Wilhite committed:
Delete NewSignView2.1.PNG

Aaron-Wilhite committed:
Add files via upload
